### PR TITLE
feat(bot): add guild count in guild-related gateway events

### PR DESCRIPTION
**Pull request by waterflamev8**
**Summary**
A clear description of what this pull request is about.
Adds guild count to the footer of the embed for "Guild Join / Leave" events

**Related issue(s)**
Provide links to relevant issues here.

**Additional context**
Add any other context or screenshots about the pull request here.

### CODE REVIEW

File: bot/core/events/handlers.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package events
 import (
 	"fmt"
 	"time"
+	"strconv"
```
Comment (chamburr) [inline bot/core/events/handlers.go:5]: nit: order imports alphabetically

File: bot/core/events/handlers.go [hunk 2]
```diff
@@ -57,12 +58,17 @@ func HandleGuildCreate(event *EventData) (err error) {
...
-	_, err = common.Session.ChannelMessageSendEmbed(config.GuildsChannel.GetInt64(), &discordgo.MessageEmbed{
+	embed := &discordgo.MessageEmbed{
 		Title:       "Guild Join",
 		Description: fmt.Sprintf("%s (%d)", guild.Name, guild.ID),
 		Color:       config.EmbedSuccessColor,
 		Timestamp:   time.Now().Format(time.RFC3339),
-	})
+	}
+	embed.Footer = &discordgo.MessageEmbedFooter{
+		Text: fmt.Sprintf("%s", strconv.Itoa(len(common.State.Guilds))),
```
Comment (chamburr) [inline bot/core/events/handlers.go]: why are you formatting a string into itself?

File: bot/core/events/handlers.go [hunk 3]
```diff
@@ -92,12 +98,17 @@ func HandleGuildCreate(event *EventData) (err error) {
...
-	_, err = common.Session.ChannelMessageSendEmbed(config.EmbedErrorColor, &discordgo.MessageEmbed{
+	embed := &discordgo.MessageEmbed{
 		Title:       "Guild Leave",
 		Description: fmt.Sprintf("%s (%d)", guild.Name, guild.ID),
 		Color:       config.EmbedErrorColor,
 		Timestamp:   time.Now().Format(time.RFC3339),
-	})
+	}
+	embed.Footer = &discordgo.MessageEmbedFooter{
+		Text: fmt.Sprintf("%s", strconv.Itoa(len(common.State.Guilds))),
```
Comment (chamburr) [inline bot/core/events/handlers.go]: same here

**Review by chamburr — CHANGES_REQUESTED** [review]

File: bot/core/events/handlers.go (hunk 2 above)
Comment (waterflamev8) [reply bot/core/events/handlers.go]: oh oops 🤦

**Review by chamburr — APPROVED** [review]

**Pull request merged**